### ISSUE TITLE
Incorrect shebang `node --harmony`

### ISSUE THREAD

**Issue by julien-f** [isRuslan/learn-generators#14]
When I try to run `learn-generator` I get the following error message:

```
/usr/bin/env: node --harmony: Aucun fichier ou dossier de ce type
```

On some systems, only one argument is allowed to be specified on the shebang directive (see [Wikipedia](https://en.wikipedia.org/wiki/Shebang_%28Unix%29#Syntax)).

I mentioned this in issue joyent/node#8338.

**Comment by isRuslan** [comment]
Oh la la, need to think about that.

**Comment by noblica** [comment]
I got the same error:
/usr/bin/env: node --harmony: No such file or directory

**Comment by isRuslan** [comment]
Should be fixed in https://github.com/isRuslan/learn-generators/commit/6fd7a5631c86e459b31d0416a55f7de03eba177d as `1.2.4` version.

@noblica @julien-f Can you update the package and check this out?

UPD: `1.2.5` version.

**Comment by isRuslan** [comment]
Hope this solved.

**Comment by julien-f** [comment]
Unfortunatly it does not work because `learn-generators.js` is not in the current working directory.

I am going to make a PR to solve this issue.

**Comment by isRuslan** [comment]
@julien-f Which version of `learn-generators` do you use?
It was fixed in #16

**Comment by julien-f** [comment]
It was fixed for global install but does not work for local install.

I think my solution is better, I am about to make my PR.
